res/values/strings.xml

```diff
@@ -681,7 +707,9 @@
     <string name="contact_setup_changed">Changed setup for %1$s.</string>
     <string name="verified_group_explain">Verified groups (experimental) provide safety against active attacks. Members are verified with a second factor by other members and messages are always end-to-end-encrypted.</string>
     <string name="copy_qr_data_success">Copied QR url to clipboard</string>
-
+    <string name="mailto_dialog_header_select_chat">Select chat to send the message to</string>
+    <string name="mailto_dialog_confirm_replace_draft">%1$s already has a draft message, do you want to replace it?</string>
```

Comment:
please add a translator comment about what `%1$s` will be. (i assume the name of the chat?)